composer.json

```diff
@@ -32,16 +35,16 @@
         "php": "^7.3 || ~8.0.0",
         "laminas-api-tools/api-tools": "^1.3",
         "laminas-api-tools/api-tools-rest": "^1.3.2",
-        "laminas/laminas-hydrator": "^2.4.2 || ^3.0",
+        "laminas/laminas-hydrator": "^2.4.2 || ^3.0 || ^4.0",
```

Comment:
Let's drop `^2` here - keeping only two majors, which is sufficient for a migration